src/tests/testUriScheme.h

```diff
@@ -35,6 +35,7 @@ class testUriScheme : public CPPUNIT_NS::TestFixture
     CPPUNIT_TEST_SUITE_END();
 
 public:
+    void setUp();
```

Comment:
Is this logic change necessary to fix build errors after sorting? If not please revert.

Reply:
It is. Changing the order of the files changes the translation unit initialization order, which means that testUriScheme gets run with uninitialized globals, failing. Initialising subsystems twice won't hurt and highlight this dependency